lib/SILOptimizer/Transforms/RedundantLoadElimination.cpp

```diff
@@ -1328,12 +1328,18 @@ SILValue RLEContext::computePredecessorLocationValue(SILBasicBlock *BB,
     Values.push_back({CurBB, LSValue::reduce(L, &BB->getModule(), LSValues, IPt)});
   }
 
+  ValueOwnershipKind mergedOwnershipKind(OwnershipKind::None);
+  llvm::for_each(Values, [&](std::pair<SILBasicBlock *, SILValue> valuePair) {
+    mergedOwnershipKind =
+        mergedOwnershipKind.merge(valuePair.second.getOwnershipKind());
```

Comment:
There is a helper on ValueOwnershipKind/OwnershipKind that does something similar to this but for normal SILValue. Please add a static function that does that instead of inlining this here.

Reply:
I used the ValueOwnershipKind version of merge that uses a range now.